drivers/hmis/lib/hud/hud_lists.json

```diff
@@ -0,0 +1,2885 @@
+[
+  {
+    "method_name": "race",
+    "name": "Race",
+    "code": "race",
+    "values": [
```

Comment:
I wonder if it's worth including some sort of description too.  I can see wanting to add hints to a bunch of these in the UI, seems like this might be the place to standardize the collection criteria.

Reply:
Hmm, I don't know about that. User-facing text about collection seems to me like it would belong in the form definition (where it's editable!), as its part of the "presentation" layer. This file here is the underlying data model, which sometimes isn't even how we choose to display the field (Race and Gender being the main two cases).

Reply:
sounds reasonable, just trying to think about how we can provide some default text without duplicating all of these keys again.